examples/assert_lockfile_up_to_date/assert_lockfile_frozen.js

```diff
@@ -0,0 +1,61 @@
+const path = require('path');
+const fs = require('fs');
+const child_process  = require('child_process');
+const readline = require('readline');
+
+const FIX_COMMAND = 'bazel run //examples/assert_lockfile_up_to_date:update_pnpm_lockfile'
```

Comment:
opt: maybe https://docs.aspect.build/aspect-build/bazel-lib/v0.12.0/docs/write_source_files-docgen.html would be a nice way to express the assertion and provides a built-in fix command